src/index.pug

```diff
@@ -0,0 +1,6 @@
+doctype.html
```

Comment:
you want to have a space between then, not a dot (.)

Reply:
Thank you very much for the observation